database/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
 
 # Create your models here.
+class Request(models.Model):
+    request_ID = models.IntegerField(max_length=50)
+    user_access_ID= models.CharField(max_length=250)
+    request_time = models.DateTimeField(max_length=100)
+
+class User(models.Model):
+    user_id = models.IntegerField(max_length=50)
+    email = models.EmailField(max_length=100)
+    password = models.CharField(max_length=250)
+    user_type = models.CharField(max_length=20)
+
+class Adviser(models.Model):
+    adviser_id = models.IntegerField(max_length=50)
+    full_name = models.CharField(max_length=250)
+    email_address = models.EmailField(max_length=100)
+    password = models.CharField(max_length=250)
+
+class Section(models.Model):
```

Comment:
Should have a [ForeignKey](https://docs.djangoproject.com/en/2.2/topics/db/examples/many_to_one/) relationship with `SchoolYear`

Reply:
done